### PR TITLE
ADCM-3098 Fix cluster bind audit

### PR DESCRIPTION
**Pull request by winmasta**

### CODE REVIEW

File: python/api/cluster/views.py [hunk 1]
```diff
@@ -228,7 +228,11 @@ class ClusterBindDetail(GenericUIView):
 
     @staticmethod
     def get_obj(kwargs, bind_id):
-        return ClusterBind.objects.get(pk=bind_id).source_service
+        bind = ClusterBind.objects.filter(pk=bind_id).first()
```
[the remark this answers is not in the document]
Comment (winmasta) [reply python/api/cluster/views.py:231]: To get cluster for audit cluster bind/unbind operations

File: python/audit/tests/test_cluster.py [hunk 2]
```diff
@@ -529,6 +529,26 @@ def test_bind_unbind(self):
             operation_type=AuditLogOperationType.Update,
         )
 
+        res: Response = self.client.delete(
```
[the remark this answers is not in the document]
Comment (winmasta) [reply python/audit/tests/test_cluster.py:532]: Fixed

**Comment by winmasta** [comment]
Bender build

**Comment by BenderArenadata** [comment]
Hey, @winmasta! Build has been launched!

**Review by a-alferov — APPROVED** [review]

**Pull request merged**